bar/90_contents/UpdateEngine/updateVEvent.js

```diff
@@ -70,16 +67,17 @@ function(request){
           body: [JSON.stringify({"error": e.message})]
         };
       }
+      for (var i = 0; i < accInfo.length; i++) {
+        if (accInfo[i].srcType == vEvent.srcType && accInfo[i].id == vEvent.srcAccountName) {
```

Comment:
Please assign vEvent.srcType & vEvent.srcAccountName and params.srcType & params.id to proper temp variable so that you can refactor properly.